tec_site/settings/base.py

```diff
@@ -0,0 +1,137 @@
+"""
+Django settings for tec_site project.
+
+Generated by 'django-admin startproject' using Django 5.0.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.0/ref/settings/
+"""
+
+from pathlib import Path
+import os
+from django.core.management.utils import get_random_secret_key
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+# SECRET_KEY = 'django-insecure-b$5++nkzv4m&=yy%n)w&qboy3fowo$9)!x^ym=m5daz_%+5i5='
+SECRET_KEY = os.getenv("DJANGO_SECRET_KEY", get_random_secret_key())
```

Comment:
Like I've mentioned, I'm not familiar with Django, but doesn't this mean this(DJANGO_SECRET_KEY) should also be part of .env.example?

Reply:
But on the other hand, we are currently not doing things that typically requires a secret key, like password hashing and signed cookies (or are we? 👀)

Reply:
You're right on both counts